docs/docs/get-started.mdx

```diff
@@ -52,7 +52,7 @@ import 'reactjs-popup/dist/index.css';
 
 ## Control Popup
 
-If you need more control over your component `reactjs-popup` we provide function as child pattern to get access to close popup action.
+If you need to have more control over your component, we provide function as child pattern to get access to close popup action.
```

Comment:
I'm really not sure what "function as child pattern to get access to close popup action." is supposed to mean. It doesn't make much sense even after looking through the source. I don't know what you're trying to say there but that should be reworded.

Reply:
I am trying to see that you can programmatically call the `close` function.